src/sentry/assistant/manager.py

```diff
@@ -0,0 +1,17 @@
+from __future__ import absolute_import
+import six
+
+
+class AssistantManager(object):
+    def __init__(self):
+        self._guides = {}
+
+    def add(self, guides):
+        for k, v in six.iteritems(guides):
```

Comment:
This can just be `self._guides = guides`. Can you also update test_assistant.py to use the manager instead? (Ideally `test_validate_guides` wouldn't need to exist since you can just make the manager's `add` method do the validation, but that has some other problems so let's put that off for later)

Also we have tests already but would also be good to test this change out in the browser to make sure things still work